.github/workflows/test.yml

```diff
@@ -55,9 +56,35 @@ jobs:
           POSTGRES_HOSTNAME: psql-a
           POSTGRES_HOSTNAME_A: psql-a
           POSTGRES_HOSTNAME_B: psql-b
+          
+  # Run unit tests on Linux Swift runners on 
+  linux-unit-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        swiftver:
+          - swift:5.2
+          - swift:5.3
+          - swift:5.4
+          - swiftlang/swift:nightly-5.5
+          - swiftlang/swift:nightly-main
```

Comment:
This has traditionally caused issues with our CI as we find that bugs in nightly cause our tests to fail and block PRs. I think it would be good to have a separate workflow that runs daily that tests the nightlies

Reply:
This is something I was working on last time I had a chance to deal with the CI for more than trivial updates, yeah.